### ISSUE TITLE
Replacer appending first entry to each entry

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ludoo0d0a/googlereaderplus#298]
```
What steps will reproduce the problem?
1. Features selected (+options for each feature):
  - Main
    - General
      - Highlight folder of current entry
  - Layout
    - Integrated Preview
      - Fullscreen preview
  - Content
    - Replacer
      - Use wedata

2. Url feed: macon.craigslist.org/rvs/index.rss

3. What is the expected output? full content for each craigslist item

4. What do you see instead? full content for each craigslist item plus the full 
content of the first item, appended to each subsequent item.  ie 

  - Item 1 
  - Item 2 + Item 1
  - Item 3 + Item 1


What is your version of GoogleReaderPlus ?
-> 3.4.0

What version of the product are you using?  (copy/paste from about:version
+ preferences/SysInfo)

Chrome (+channel): stable
Screen resolution:
SysInfo:


On what operating system - Windows(XP, Vista...), Linux(Ubuntu, Debian...),
MacOs(9, X...)? Win7

Which others extensions are installed ?


Please provide any additional information below:

Replacer entry is:
images from craigslist
craigslist.org
css:#userbody
$0

I spent quite a while playing with this, using Inspect Element, and so on to 
figure out what was going on.  Seems to be happening because when reader 
initially loads, it only loads four items.  I had noticed that for the first 
couple entries, Replacer seemed to work as intended.  As I scrolled through the 
items, Replacer would start appending the very first item to the bottom of each 
new entry.  Going back to where I stated that reader only loads four items on 
initial load, this coincides with my observations, those first four load 
properly with Replacer "replacing" the content properly.  After that first four 
though, as I scroll through the items, and reader is "adding" the "new" item, 
it appears that Replacer replaces the new item properly as the now first item, 
but is still holding the content from the original first item and appends that 
to the "new first item"

In other words, it seems that Replacer is iterating through the list of items 
loaded in reader(which seems to initially only be four items), and works 
properly.  As I scroll through the items, reader is making requests for the 
next item in the list, adding to the initial list of items Replacer iterated 
through.  Replacer replaces the next item properly, but is appending or 
concatenating the original first item to the end of the new first item.  If 
using an array "item"  (item[0]=first item) then when item five is added by 
reader (item[4]=item five + first item) is what Replacer is outputting.

I sure hope this makes some sort of sense.
```

Original issue reported on code.google.com by `phillip....@gmail.com` on 3 Sep 2010 at 5:28
